curriculum/challenges/english/02-javascript-algorithms-and-data-structures/object-oriented-programming/inherit-behaviors-from-a-supertype.english.md

```diff
@@ -32,6 +32,10 @@ tests:
     testString: assert(typeof duck !== "undefined", 'The <code>duck</code> variable should be defined.');
   - text: The <code>beagle</code> variable should be defined.
     testString: assert(typeof beagle !== "undefined", 'The <code>beagle</code> variable should be defined.');
+  - text: The <code>duck</code> variable should be initialised with <code>Object.create</code>.
+    testString: assert(/duck\s*=\s*Object\.create\(\s*Animal\.prototype\s*\);/.test(code), 'The <code>duck</code> variable should be initialised with <code>Object.create</code>');
```

Comment:
The **";"** at the end of /duck\s*=\s\*Object\\.create\\(\s*Animal\\.prototype\s\*\\)**;** is not compulsory in JavaScript. We can omit the semi-colon.  

Let us make the same change for beagle as well

Reply:
@rupali317 Even though in this case a semi-colon is not needed, it is still considered a best practice and we teach using semi-colons through the curriculum.

Reply:
After thinking about that further, with the optional semi-colon it would then require more checks to make sure no chaining, references, etc. are used after the object is created (although subsequent tests would then fail anyway)
Something like this perhaps
`/\sduck\s*=\s*Object\s*\.\s*create\s*\(\s*Animal\s*\.\s*prototype\s*\)\s*[^()[\]{}.]\s*;?$/m`

Reply:
Actually, after thinking about it, that expression would accept incorrect answers, as it doesn't make sure another method is not chained afterwards, it's not curried, etc.

Reply:
@RandellDawson Yes, we do use semicolons throughout the curriculum but in tests we don't enforce the semicolons. For example other [this](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-export-to-reuse-a-code-block.english.md) challenge. 
Enforcing semicolons *now* would be inconsistent with the other challenges/tests.

Reply:
@ezioda004 You are correct.  I was thinking about semi-colons in the description sections.

Reply:
@RandellDawson We should revert the semicolon part as it'd restrict the solution style.

Reply:
I will take care of it first thing tomorrow.

Reply:
I got free time :)

Reply:
Please go ahead then :)